integration/spark/app/src/main/java/io/openlineage/spark/agent/lifecycle/SparkSQLExecutionContext.java

```diff
@@ -71,7 +71,9 @@ public SparkSQLExecutionContext(
 
   @Override
   public void start(SparkListenerSQLExecutionStart startEvent) {
-    log.debug("SparkListenerSQLExecutionStart - executionId: {}", startEvent.executionId());
+    if (log.isDebugEnabled()) {
```

Comment:
I don't like the redundant approach of checking logging level in our code. This should happen within logger library. 
Extra condition makes only sense when resolving `debug` method parameters can be expensive like json. Otherwise I don't think it's necessary. 
I wouldn't do that for `getClass` and `getContainerName` as well.